### ISSUE TITLE
ASYNC POLL output invisible due to missing color contrast

### ISSUE THREAD

**Issue by jojoob** [ansible-semaphore/semaphore#1135]
The output for async poll tasks is invisible because of black text color on black background (independent from dark mode switch):
![Screenshot from 2023-01-03 09-03-01](https://user-images.githubusercontent.com/4180675/210354683-5e11b7ce-80b2-4aa1-bac8-f12f9c28f5bc.png)

The HTML structure for this kind of output looks a bit different:
![Screenshot from 2023-01-03 13-11-30](https://user-images.githubusercontent.com/4180675/210354999-e1e11c53-64fe-4975-9ba9-3da489ec1a80.png)

I worked around with this local CSS override:
```css
.task-log-records__record .task-log-records__output b span {
  color: grey !important;
}
```
